backend/src/main/java/ai/giskard/web/dto/ApplyPushDTO.java

```diff
@@ -0,0 +1,24 @@
+package ai.giskard.web.dto;
+
+import ai.giskard.worker.CallToActionKind;
+import ai.giskard.worker.PushKind;
+import com.dataiku.j2ts.annotations.UIModel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.Map;
+import java.util.UUID;
+
+@Getter
+@Setter
+@UIModel
+@NoArgsConstructor
+public class ApplyPushDTO {
+    private UUID modelId;
+    private UUID datasetId;
+    private int rowIdx;
+    private PushKind pushKind;
```

Comment:
Since PushKind and CallToActionKind is a MLWorker generated type we have `any` type in ts due to `missing translation`.  @andreybavt do you think there is a quick fix or we can keep like this.